### ISSUE TITLE
Mimetypes: Lossless audio files

### ISSUE THREAD

**Issue by hlechner** [numixproject/numix-icon-theme#595]
Who has a good headphone and possible a good sound card probably love to listen to the music in its original form instead of lossy data compression like mp3 and others.

So a different mimetype for lossless music file is extremely appreciated by music lovers like me, even if it'll be just a different background color.

Ps: Yea, I know, it's not priority and I understand it, so don't worry and thanks anyway.

![image](https://cloud.githubusercontent.com/assets/11244067/8270453/3814b2ec-17b6-11e5-9789-878c4218e1b4.png)

**Comment by hlechner** [comment]
I'm not so creative, the only things that I can imagine:
- maybe add a ribbon symbol, like this one:

![image](https://cloud.githubusercontent.com/assets/11244067/8445141/afd34f76-1f6f-11e5-9520-65ddd14d0437.png)

- Maybe a medal style

**Comment by hlechner** [comment]
Hey guys @Foggalong @palob @ChuckDaniels87 @wa4557 could you guys please give me some ideas on this one?

I know, I avoid to mention people like that, it's just because I'm totally stuck on this for months and I really trying a lot but nothing comes to my mind even that I made a big research about it.

**Comment by Foggalong** [comment]
Not especially keen on adding those wave things to the icon

**Comment by hlechner** [comment]
that are just an old drafts, that i made just to try something and maybe progress from it for another design idea.

However I couldn't find a better design idea yet. the old drafts are not supposed to be merged anyway.

**Comment by ChuckDaniels87** [comment]
Different colour and extension label? Actually, there are all audio formats, both lossless and compressed. You can assign one colour to compressed and another one to lossless.

**Comment by palob** [comment]
Yeah, text labels sounds like the easiest yet not the most elegant solution. A different colour isn't very self-evident.

**Comment by ChuckDaniels87** [comment]
I mean both of them.

**Comment by Foggalong** [comment]
Yeah, I'm game for both :)

**Comment by palob** [comment]
So should we add text labels for all audio/video mimetypes eventually?

**Comment by hlechner** [comment]
Thanks @ChuckDaniels87 good idea

and guys, which color should I use? Maybe just a different tone?

Oh, now I remembered a possible problem, we normally use the text to show the extension, however there are some mimetypes that each can have different extensions. So in that case what we should do? or maybe just skip these ones?

**Comment by palob** [comment]
Well, the JPG-labelled mimetype for instance is used for the extensions .jpg .jpeg .jpe ....
TGZ is .tar.gz....
Is there one charecteristic abbreviation?

**Comment by hlechner** [comment]
Yea, I think it's possible to use the most common type of the mime, for example the mp4a is a apple lossless but it's recognized as normal mp4 due to format used by the file. (this lossless I didn't add in the list for this reason)

**Comment by palob** [comment]
Ah OK, so it's not that easy and there might arise confusion in some cases.

**Comment by palob** [comment]
Maybe a different colour (a bit darker/lighter) isn't that bad a solution.

Or how a about a different colour for the note or a semiquaver note?

**Comment by hlechner** [comment]
Original one:
![image](https://cloud.githubusercontent.com/assets/11244067/17755082/4d42070a-64ae-11e6-96f1-2b23784265ce.png)

-- drafts:
With semiquaver note symbol:
![image](https://cloud.githubusercontent.com/assets/11244067/17755104/67eb652e-64ae-11e6-98a0-14a6fa2d6534.png)

Darker (base color/symbol):
![image](https://cloud.githubusercontent.com/assets/11244067/17755134/905df17a-64ae-11e6-8183-f475fab0caa7.png)![image](https://cloud.githubusercontent.com/assets/11244067/17755138/9470b2c0-64ae-11e6-82fd-79e97d9a5321.png)

What do you think @palob ?

**Comment by palob** [comment]
Is there a way to have the semiquaver more in line with the existing note design?

**Comment by hlechner** [comment]
I'm trying to adapt the symbol, but until now, not good results so far.

If you know any design as reference please let me know.
